generator-jsonforms/package.json

```diff
@@ -17,9 +17,11 @@
   },
   "dependencies": {
     "@types/yeoman-generator": "^3.0.1",
+    "@jsonforms/core": "^2.0.12",
     "chalk": "^2.4.2",
     "clear": "^0.1.0",
     "figlet": "^1.2.1",
+    "jsonforms-basic-project": "git+https://git@github.com/roxhens/jsonforms-yo-basic-project.git",
```

Comment:
The referenced repo needs to be moved to our organization. But on a different note: is this the same template you use for bootstraping with OpenAPI? There are commits on the referenced project which remove the submit button and so on. There should be two templates one for the 'basic' use case and another for the OpenAPI one, but it seems there's only tempalte proejct. Or am I missing something here?

Reply:
I updated the url with an eclipsesource repo. Concerning the commits that remove the submit button, they are not present in the new repo.